api_server/src/main/java/io/zetch/app/domain/review/ReviewDto.java

```diff
@@ -0,0 +1,17 @@
+package io.zetch.app.domain.review;
+
+import io.zetch.app.domain.location.LocationDto;
+import io.zetch.app.domain.user.UserDto;
+import java.io.Serializable;
+import lombok.*;
+
+@Value
+@Builder
+@AllArgsConstructor
+public class ReviewDto implements Serializable {
```

Comment:
I believe all the dtos have the id field removed.

Reply:
I think every Dtos except `Post` should have ids. Without knowing Ids using APIs, we cannot create joined instances.